frontend/src/containers/HomePage.js

```diff
@@ -13,9 +12,31 @@ import CollectiveCard from '../components/CollectiveCard';
 import fetchHome from '../actions/homepage/fetch';
 
 const mapCollectiveCardProps = group => {
-  group.sponsorCount = filterCollection(group.backers, {tier: 'sponsor'}).length;
-  group.backersCount = filterCollection(group.backers, {tier: 'backer'}).length;
-  group.monthlyIncome = (group.yearlyIncome / 12);
+  group.mission = `We are on a mission to ${group.mission}`;
+  group.stats = [];
+
+  if(group.contributors && Object.keys(group.contributors).length > 0)
```

Comment:
it is already. https://github.com/OpenCollective/eslint-config-opencollective/blob/master/index.js. We need to upgrade it in package.json.